.changeset/strong-tomatoes-watch.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
please delete this file. changesets are not required for documentation changes and certainly should result in new major versions being cut for each file